lib/screens/search_screen.dart

```diff
@@ -0,0 +1,102 @@
+import 'package:auto_route/annotations.dart';
+import 'package:e_commerce_application/provider/search_screen_provider.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import '../provider/home_screen_provider.dart';
```

Comment:
Update to package imports. Do this throughout project for any imports using relative path